locale/translations/fr_FR.json

```
@@ -47,6 +48,13 @@
     "menu.api_keys": "Clés d'API",
     "menu.create_api_key": "Créer une nouvelle clé d'API",
     "menu.shared_entries": "Articles partagés",
+    "menu.sort_feeds_sort": "Ordre",
+    "menu.sort_feeds_name": "nom",
+    "menu.sort_feeds_published": "heure publiée",
+    "menu.sort_feeds_last_check": "Dernière vérification",
+    "menu.sort_feeds_unread": "non lues",
```

It depends what we're talking about: what is read/unread?
```suggestion
    "menu.sort_feeds_unread": "non lu",
```